canvas_cli/templates/plugins/default/{{ cookiecutter.project_slug }}/README.md

```diff
@@ -8,11 +8,14 @@
 
 ```
 {{ cookiecutter.project_slug }}/
-├── commands/
-├── content/
-├── effects/
-├── protocols/
-├── views/
+├── {{ cookiecutter.project_slug }}/
```

Comment:
We don't want this extra nesting, the CANVAS_PLUGIN_TAGS, or the pyproject.toml

Reply:
i think @beaugunderson and @jrwils  had to add back in the pyproject.toml and the nesting for the install to work. i'll let them weigh in on why!

Reply:
we did for the demo but yeah we want to rip that all out

Reply:
ok good to know, i can make that change!